osu.Game.Rulesets.Taiko/osu.Game.Rulesets.Taiko.csproj

```diff
@@ -15,4 +15,8 @@
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\osu.Game\osu.Game.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="MathNet.Numerics" Version="5.0.0" />
+  </ItemGroup>
```

Comment:
I'm very hesitant about this. The library is 1.6mb, and it also hides the implementation (and potential complexity of it).

Looking into the underlying code, I don't think it's complex enough that we can't have a local implementation if this is required.

Reply:
Don't worry about doing this just now. I realised that the other PR is also using this.

We'll handle pulling the function out if/when this needs to be merged.

Reply:
Sounds good to me either way, the package will be necessary for potential future reworks (root finding for example) but for the purposes of this rework it can be locally implemented

Reply:
Recent changes have added many more concepts from mathnet, so unless you have any further notes I feel fine with marking this resolved